api-tester/src/main/java/com/revenuecat/apitester/java/StoreTransactionAPI.java

```diff
@@ -13,7 +13,6 @@
 final class StoreTransactionAPI {
     static void check(final StoreTransaction transaction) {
         final String orderId = transaction.getOrderId();
-        final List<String> skus = transaction.getSkus();
```

Comment:
i think we've generally left deprecated stuff in the api testers, since it's technically still public

could be convinced otherwise, but we should be consistent either way